pkg/reconciler/pipelinerun/resources/resultrefresolution_test.go

```diff
@@ -26,6 +24,7 @@ var (
 		Type:   apis.ConditionSucceeded,
 		Status: corev1.ConditionFalse,
 	}
+	_ = failedCondition
```

Comment:
could you help me understand what is this alias for?

Reply:
This variable is not used, but I don't want to delete it so I just left it like this.
I delete this variable directly to avoid ambiguity.